README.md

```diff
@@ -176,6 +176,10 @@ c.post('/search', Document(text='request(on=something)'), on_done=print_matches)
     - [Minimum Working Example](.github/2.0/cookbooks/Serving.md#minimum-working-example)
     - [`Flow`-as-a-Service](.github/2.0/cookbooks/Serving.md#flow-as-a-service)
     - [Deliver OAS3.0 Friendly API](.github/2.0/cookbooks/Serving.md#extend-http-interface)
+- ☁️ `JinaD`: manage & deploy Jina on remote
+  - [Server & Client](.github/2.0/cookbooks/Daemon.md#jinad-daemon)
+  - [Remote Executors](.github/2.0/cookbooks/Daemon.md#restful-executors)
+  - [Remote Flows](.github/2.0/cookbooks/Daemon.md#restful-flows)
 - 📓 [Developer Reference](https://docs.jina.ai)
 - 🧼 [Clean & Efficient Coding in Jina](.github/2.0/cookbooks/CleanCode.md)
```

Comment:
Just to be clear, I'm fine with "&" in bullet links (keeps them short). Only in paragraphs let's expand to "and"